e2e/support/helpers/e2e-misc-helpers.js

```diff
@@ -175,6 +175,12 @@ export function visitDashboard(dashboard_id, { params = {} } = {}) {
       validQuestions = validQuestions.filter(
         card => card.dashboard_tab_id === params.tab,
       );
+    } else if (tabs.length > 0) {
+      // if dashboard has tabs, but there is no tab param, default to using the first tab
+      const firstTab = tabs[0];
+      validQuestions = validQuestions.filter(
+        card => card.dashboard_tab_id === firstTab.id,
+      );
     }
```

Comment:
@nemanjaglumac @EmmadUsmani check this out.

Visit dashboard wasn't working with instance analytics dashboards because they have tabs now, and we were awaiting all the dashcards to load, but only some of them load because we only load the first tab.

I added logic to visitDashboard such that if the dashboard has tabs, and you don't pass a specific tab id param, then we just expect the first tab to be loaded.

The pattern of setting a specifc tab id is kinda fragile, and hard to use since they're globally unique - so you can't even visit tab 1 on every dashboard. I wonder if we should remove this tabs param altogether?

Reply:
I really really really don't think we should mess with a global helper that's used in hundreds of tests here.
Can that be a separate PR, pretty please?

Reply:
sure